spec/index.bs

```diff
@@ -624,12 +622,37 @@ interface GPURenderPassEncoder : GPUProgrammablePassEncoder {
     void drawIndexed(u32 indexCount, u32 instanceCount, u32 firstIndex, i32 baseVertex, u32 firstInstance);
 
     // TODO add missing commands
+}
+
+interface GPURenderPassEncoder : GPURenderEncoderBase {
+    void executeBundle(sequence<GPURenderBundle> bundles);
+
+    void endPass();
+};
+
+dictionary GPURenderBundleDescriptor : GPUObjectDescriptorBase {
+    //TODO: reusability flag?
```

Comment:
This would be a wrong place for this flag, since in Vulkan we need to specify `VK_COMMAND_BUFFER_USAGE_ONE_TIME_SUBMIT_BIT` at the beginning of recording, not the end.
Perhaps, we don't need this structure at all? Why not require the user to specify all the info we need at the creation time instead of deferring to this moment?

Reply:
Sorry this was a leftover from copy pasting `GPUCommandEncoderDescriptor`. Removed. Agreed that the reusable flag (if bundles aren't always reusable) should be on the `GPURenderBundleEncoderDescriptor`.